source/includes/_themes_converting.md

```diff
@@ -0,0 +1,295 @@
+# Converting themes
+
+Earlier Mautic templates used PHP based on Symfony templates, however with the release of the 2.0 series, templates now use Twig.
+
+There are some changes that you will need to make to your older templates to have them work with the newer versions of Mautic.
+
+## Directory structure
+
+Firstly, the structure of the directory and the files required have changed:
+
+### Mautic 1.x theme structure:
+
+themes/HelloWorld/
+```
+- - config.php
+- - html/ 
+- - - - - base.html.php
+- - - - - email.html.php
+- - - - - form.html.php
+- - - - - message.html.php
+- - - - - page.html.php
+```
+
+### Mautic 2.x theme structure:
+
+themes/HelloWorld/
+```
+- - config.json
+- - thumbnail.png
+- - html/ 
+- - - - - base.html.twig
+- - - - - email.html.twig
+- - - - - form.html.twig
+- - - - - message.html.twig
+- - - - - page.html.twig
+```
+
+## JSON configuration file
```

Comment:
Add a note here that there is a command line tool to convert an existing php theme config into json, `mautic:theme:json-config --theme="THEME_NAME"`